aeron-test-support/src/main/java/io/aeron/test/cluster/TestCluster.java

```diff
@@ -2137,6 +2139,7 @@ public static final class Builder
         private File markFileBaseDir = null;
         private String clusterBaseDir = System.getProperty(
             CLUSTER_BASE_DIR_PROP_NAME, CommonContext.generateRandomDirName());
+        private String aeronBaseDir = CommonContext.AERON_DIR_PROP_DEFAULT;
```

Comment:
This should be replaced with `CommonContext.generateRandomDirName()` to ensure that tests don't interact badly.